tests/fixtures/config/minimal_contenttypes.yaml

```diff
@@ -0,0 +1,11 @@
+# Minimal Contenttypes
```

Comment:
Content Types*
also change file name

Reply:
why change the filename?

Reply:
because it uses snake case, so it can be `minimal_content_types.yaml`

Reply:
We've always used `contenttypes.yml` in `config/` though.. I've changed it, but i'm not sure I like the inconsistency. 🤔

Reply:
This is a "bc break" between 3 and 4 ;) Acually I prefer to call it `Content Definitions` but types is shorter ;)